config/boards/ayn-odin2.conf

```diff
@@ -0,0 +1,83 @@
+# Generate kernel and rootfs image for Qcom ABL booting
+declare -g BOARD_NAME="Ayn Odin2"
+declare -g BOARD_MAINTAINER="FantasyGmm"
+declare -g BOARDFAMILY="qcom-abl"
+declare -g KERNEL_TARGET="sm8550"
+declare -g KERNELPATCHDIR="arm64-sm8550"
+declare -g EXTRAWIFI="no"
+declare -g BOOTCONFIG="none"
+declare -g IMAGE_PARTITION_TABLE="gpt"
+declare -g -a ABL_DTB_LIST=("qcs8550-ayn-odin2-hypdtbo")
+
+# Use the full firmware, complete linux-firmware plus Armbian's
+declare -g BOARD_FIRMWARE_INSTALL="-full"
+
+function post_family_config_branch_sm8550__edk2_kernel() {	
+	declare -g KERNELSOURCE='https://github.com/edk2-porting/linux-next'
+	declare -g KERNEL_MAJOR_MINOR="6.7" # Major and minor versions of this kernel.
+	declare -g KERNELBRANCH="branch:integration/ayn-odin2"
+	declare -g LINUXCONFIG="linux-${ARCH}-${BRANCH}" # for this board: linux-arm64-sm8550
+	display_alert "Setting up kernel ${KERNEL_MAJOR_MINOR} for" "${BOARD}" "info"
+}
+
+function ayn-odin2_is_userspace_supported() {
+	[[ "${RELEASE}" == "trixie" ]] && return 0
+	[[ "${RELEASE}" == "noble" ]] && return 0
+	return 1
+}
+
+function post_family_tweaks__ayn-odin2_enable_services() {
+	if ! ayn-odin2_is_userspace_supported; then
+		if [[ "${RELEASE}" != "" ]]; then
+			display_alert "Missing userspace for ${BOARD}" "${RELEASE} does not have the userspace necessary to support the ${BOARD}" "warn"
+		fi
+		return 0
+	fi
+
+	# We need unudhcpd from armbian repo, so enable it
+	mv "${SDCARD}"/etc/apt/sources.list.d/armbian.list.disabled "${SDCARD}"/etc/apt/sources.list.d/armbian.list
+
+	# Add zink env
+	echo '__GLX_VENDOR_LIBRARY_NAME=mesa' | tee -a "${SDCARD}"/etc/environment
+	echo 'MESA_LOADER_DRIVER_OVERRIDE=zink' | tee -a "${SDCARD}"/etc/environment
+	echo 'GALLIUM_DRIVER=zink' | tee -a "${SDCARD}"/etc/environment
+
+	do_with_retries 3 chroot_sdcard_apt_get_update
+	display_alert "$BOARD" "Installing board tweaks" "info"
+	do_with_retries 3 chroot_sdcard_apt_get_install alsa-ucm-conf unudhcpd mkbootimg git
+
+	# Disable armbian repo back
+	mv "${SDCARD}"/etc/apt/sources.list.d/armbian.list "${SDCARD}"/etc/apt/sources.list.d/armbian.list.disabled
+	do_with_retries 3 chroot_sdcard_apt_get_update
+
+	if [[ "${RELEASE}" == "noble" ]];then
```

Comment:
I think trixie should also have packages below.

Reply:
ok，i will delete this